### ISSUE TITLE
When i open 2 accounts, 1 account becomes unresponsive and I have to force close

### ISSUE THREAD

**Issue by Drogon1234** [Healix/Gw2Launcher#47]
This has been happening for both my boyfriend and I since this week. I'm not sure if any of the patches messed anything up?

We both do leyline anomaly on 2 accounts at a time and we can login on both accounts. But every time the leyline is up and I try to swap to my 2nd account it just won't open anymore and i have to force close GW2.

We had no issues with this before.

**Comment by Healix** [comment]
Which account freezes, and does this happen when you go to launch the second account, or was it already started (at least to character select). Is this only a problem at the leyline?

If you're using d912pxy, it can cause a 2nd account to freeze until the first is exited. Use "localized execution" set to "full" to prevent that (settings > guild wars 2 > management).

If both accounts were already started and one freezes, ensure your system has the resources to play 2 accounts - verify the game's graphics settings, especially if this only a problem at a high population event. When low on memory, it'll be harder to swap back to an account that was minimized (it can take 10~60 seconds or more).

**Comment by Drogon1234** [comment]
Both accounts will already be logged in and standing somewhere in the map where the ley line is about to spawn. The one that freezes is the one that will be in the background at the time (the one I will not be actively playing and is just standing around at a waypoint).

I am not using d912pxy and all accounts have the lowest graphics settings on everything. The only thing is that they will be on fullscreen, because if i switch it to borderless windowed it will be reset to fullscreen the next time i open the account.

Our hardware hasn't changed and we've been doing this for over a month but this issue only showed up this week.

**Comment by Healix** [comment]
GW2 is in read-only mode when you launch it (through Gw2Launcher). If you want to change the graphics, sound or login settings in game, you have to right click > selected > launch (normal). For graphics settings specifically, you can right click > edit > local.dat > graphics. 

Full screen mode is worse off when switching, especially since minimizing the game "pauses" it - it loads everything when you switch back to it.

As for low resources, it's just something to keep an eye on - ctrl+shift+esc to open the task manager - check what's happening when it's frozen.

**Comment by Drogon1234** [comment]
If i do right click > edit > local.dat > graphics and put an account to windowed fullscreen and click on ok, when i launch the game it will still launch in fullscreen mode and if i go back to the settings in the launcher it is also back to fullscreen.

Any idea how i can fix this? I have tried to open the launcher as admin and do it then, but it will still change it back to fullscreen in the settings once i open the game.

**Comment by Healix** [comment]
If you change the graphics options in Gw2Launcher, OK, then go back into the settings (without launching), is it still changed? Try changing the animation option, OK, then launch it and check if it was applied in the game. This is to simply test if GW2 is partially or entirely ignoring these settings. Note you can't change these settings while an account is active.

Right click > Selected > Launch (normal) - you should now be able to change the graphics options from within the game itself.

**Comment by Drogon1234** [comment]
If I click OK and then go back into the settings it is still set to windowed fullscreen, but as soon as I launch the game, it puts it back to fullscreen. I have tried changing the animation settings, and that one does stay the same. For example, I turned it from low to high and when I launched the game it launched it with high.

However, I have now tried to Launch (normal) and change the setting in-game and that does seem to have done the trick. It is now also staying on windowed fullscreen when I don't open it as Launch (normal).

Sadly this doesn't seem to have fixed the overall issue, as I just did leyline with 2 accounts both playing windowed fullscreen and the account in the background froze yet again.
If no one else is reporting the issue, I guess it is not coming from the launcher. Although I find it very strange that it is happening to both my partner and I and we are running it on different hardware and had no issues prior to this week.

Feel free to close the issue, unless you have any other ideas I can try to not have the account in the background freeze. I may try to run it with 2 accounts in windowed mode tomorrow.
